.github/workflows/manual.yml

```diff
@@ -0,0 +1,55 @@
+name: "Build and deploy documentation"
+
+on:
+  pull_request: # testing workflow
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      # build the manuals only when docs directory is updated
+      - docs/**
+      - modules/**
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: write
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  check_date:
+    runs-on: ubuntu-latest
+    name: Check latest commit
+    outputs:
+      should_run: ${{ steps.should_run.outputs.should_run }}
+    steps:
+      - uses: actions/checkout@v4.1.7
+      - name: print latest_commit
+        run: echo ${{ github.sha }}
+
+      - id: should_run
+        continue-on-error: true
+        name: check latest commit is less than a day
+        if: ${{ github.event_name == 'schedule' }}
+        run: test -z $(git rev-list --after="24 hours" ${{ github.sha }}) && echo "::set-output name=should_run::false"
+
+  publish:
+    needs: check_date
+    if: ${{ needs.check_date.outputs.should_run != 'false' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.7
+      - uses: DeterminateSystems/nix-installer-action@main
+      - run: |
+          nix build .#docs -Lv
+          cp -r result public
```

Comment:
`nix build -Lv .#docs -o `public` unless you need to write to the directory. `-o` stands for `--out-link`